components/delete_channel_modal/delete_channel_modal.jsx

```diff
@@ -59,19 +58,12 @@ export default class DeleteChannelModal extends React.PureComponent {
         this.setState({show: false});
     }
 
-    handleKeyDown(e) {
```

Comment:
I don't think we should remove this. I'll see what I can do to avoid deleting a channel on `enter` key press whenever the tab focus is on modal's `x` or close button.

Reply:
I think that setting the react `autoFocus` attribute on the button should keep the hitting enter functionality when the modal is opened.

Reply:
Looks like it's not really needed. I just added the `autoFocus` as you mentioned.